app_flutter/lib/status_grid.dart

```diff
@@ -24,14 +25,16 @@ class StatusGrid extends StatelessWidget {
     return SingleChildScrollView(
```

Comment:
You could test that the height of this in the widget tester is the window height less 56, or similar.

Reply:
Since I just used the Expanded widget, should this still be tested?

Reply:
Should be good.